cfme/tests/automate/test_service_automate.py

```diff
@@ -30,11 +29,8 @@ def new_users(appliance):
 
     yield users
     for user in users:
-        if not BZ(1720273).blocks:
-            user.delete_if_exists()
-        else:
-            user = appliance.rest_api.collections.users.get(name=user.name)
-            user.action.delete()
+        user = appliance.rest_api.collections.users.get(name=user.name)
```

Comment:
This is a little out of scope of this PR, but how about using [`users`](https://github.com/ManageIQ/integration_tests/blob/master/cfme/rest/gen_data.py#L466) to create users? You'll probably need to change the group in `gen_data.users`, but then you will not need to delete the users that you create. You will only need to instantiate those users.